### DISCUSSION TITLE
Copy Button

### DISCUSSION THREAD

**Discussion by RobioEpo** [Etar-Group/Etar-Calendar#1043] [Ideas]
Some form of button to be able to copy an event, would be very much appreciated. 

![signal-2021-11-20-22-18-53-875](https://user-images.githubusercontent.com/49530442/142752018-9965eb57-01eb-4a87-bb8f-ad2b1860fe83.jpg)

**Comment by s-k-e-t-c-h** [comment]
Latest release #1284 brings us this feature.
And I love it.
